### ISSUE TITLE
Informal review comments

### ISSUE THREAD

**Issue by beverloo** [beverloo/periodic-background-sync#8]
Review comments for the current explainer -

  1. The a/b/c examples in both (1) and (2) of the "use cases" section should be lists.
  1. Example 1b. of the "use cases" feels like content to me. Perhaps pushing a critical application update might be a better example?
  1. I'm not sure what to make of the "...and is GDPR compliant" addition in "use cases".
  1. In "non-goals", there can be multiple periodic tasks per origin w/ varying frequency, right?
  1. In "security and privacy", I'd rephrase the "run forever" bit. It just makes it sound overly scary.

Open questions -

  1. The previous explainer contained conditionals for power & network state. Why doesn't this proposal?
  1. Do the tags of One-shot Background Sync and Periodic Background Sync share a namespace?
  1. What are the timing constraints? There are a number of suggestions ("can limit the number", "has the final timing decision"), but it might be worth adding a section specifically to discuss this.

Please also ask @jakearchibald to take a look.

**Comment by mugdhalakhani** [comment]
Regarding review comment 4, what I was trying to convey was that the developer can request different tasks at different frequency for the same origin, but the browser might end up running them all at the same frequency. I've made the wording clearer now, hopefully.
